src/components/composer/formula_assistant/formula_assistant.ts

```diff
@@ -50,6 +51,15 @@ export class FunctionDescriptionProvider extends Component<Props, SpreadsheetChi
     functionDescription: Object,
     argsToFocus: Array,
   };
+  static components = { Collapse };
+
+  private state: { opened: boolean } = useState({
```

Comment:
I would rename the boolean to isOpen. It's always clearer in my mind when booleans start by is... :D